app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -90,7 +89,7 @@ Visit the UK government’s official [Teaching Vacancies](https://teaching-vacan
 #### Step 2
 Apply for a teaching job with a school which can sponsor your visa. Your salary must meet the requirements for a Skilled Worker visa. 
 
-[Learn more about applying for a teaching job in England](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
+Learn more about [applying for a teaching job in England](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
```

Comment:
This is the same link as the `$teachinuk$` link - should we have them both?

Ideally we shouldn't have links to the same page with different link text?